bxml/buildingBlocks/credentials.md

```diff
@@ -0,0 +1,18 @@
+## Adding Credentials
+
+Your credentials are required to make any requests to the Bandwidth API. **Never publish your credentials.** Your credentials are unique to your account and allow Bandwith to verify you are the one making the requests.
+
+Copy and paste the following code block and replace "Your_..." with the appropriate credentials.
+
+```js
+var client = new Bandwidth({
+    userId    : "YOUR_USER_ID",
+    apiToken  : "YOUR_API_TOKEN",
+    apiSecret : "YOUR_API_SECRET"
+});
+```
```

Comment:
Instead of putting the credentials **IN** the code here, replace these with the standard environment variables and add a section on how to set environment vars like [here](https://github.com/dtolb/bandwidth-call-screen#environment-variables)

```js
var client = new Bandwidth({
  userId    : process.env.BANDWIDTH_USER_ID,
  apiToken  : process.env.BANDWIDTH_API_TOKEN,
  apiSecret : process.env.BANDWIDTH_API_SECRET
};
```

this will encourage our users to follow best practices 😎